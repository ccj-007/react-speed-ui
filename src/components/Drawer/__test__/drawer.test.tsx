import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Drawer } from '../../index';

describe('test Drawer component', () => {
  it('should render', () => {
    const { asFragment } = render(<Drawer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
