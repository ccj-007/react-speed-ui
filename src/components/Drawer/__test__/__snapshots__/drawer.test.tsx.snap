// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Drawer component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-drawer speed-drawer-left css-7zjjat-Drawer"
    style="left: -200px; width: 200px; min-height: 100vh;"
  >
    <div
      class="speed-drawer-header"
    >
      <div
        class="speed-drawer-close"
      >
        X
      </div>
    </div>
    <div
      class="speed-drawer-content"
    />
    <div
      class="speed-drawer-footer"
    >
      <button
        class="speed-button speed-button-primary css-1dis8g1-Button"
        data-testid="button"
      >
        确认
      </button>
      <button
        class="speed-button speed-button-primary css-1dis8g1-Button"
        data-testid="button"
      >
        取消
      </button>
    </div>
  </div>
</DocumentFragment>
`;
